db/schema.rb

```diff
@@ -1251,7 +1255,18 @@
        LANGUAGE plpgsql
       AS $function$
                   BEGIN
-                    new.phone_index = (regexp_replace(new.phone, '\\D','', 'g'));
+                    new.phone_index = (regexp_replace(new.phone, '\D','', 'g'));
```

Comment:
@wwahammy, what did you do to avoid this change on your PR?

Reply:
I just had to manually change it back. I need to report the bug to fx